rustls/src/conn.rs

```diff
@@ -958,6 +958,12 @@ impl ConnectionCommon {
             .prepare_message_decrypter(dec);
     }
 
+    #[cfg(feature = "quic")]
+    pub fn missing_extension(&mut self, why: &str) -> Error {
+        self.send_fatal_alert(AlertDescription::MissingExtension);
+        Error::PeerIncompatibleError(why.to_string())
```

Comment:
Seems like peer misbehavior, since the extensions are required for QUIC to work, right?

Reply:
Yup, copy/pasta error.